opcua/server/history.py

```diff
@@ -130,7 +310,7 @@ def event_notification(self, event):
 class HistoryManager(object):
     def __init__(self, iserver):
         self.iserver = iserver
-        self.storage = HistoryDict()
+        self.storage = HistorySQLite()  # HistoryDict() HistorySQLite()
```

Comment:
Sorry that was left over from my testing and I didn't know how you wanted to organize other history interfaces.

Sorry again, I'm bad at programming. I will try to figure out how to make some tests for this.